.github/workflows/vercel-preview-rewise.yaml

```
@@ -1,74 +1,45 @@
-
-      - name: Check for newer commits
-        id: check_commits
-        run: |
-          CURRENT_BRANCH=${GITHUB_REF#refs/heads/}
-          git fetch origin $CURRENT_BRANCH || { echo "Remote branch not found, continuing with build"; echo "skip_build=false" >> $GITHUB_OUTPUT; exit 0; }
-          LATEST_COMMIT=$(git rev-parse origin/$CURRENT_BRANCH 2>/dev/null || echo "")
-          CURRENT_COMMIT=${GITHUB_SHA}
-          if [ -n "$LATEST_COMMIT" ] && [ "$LATEST_COMMIT" != "$CURRENT_COMMIT" ]; then
-            echo "Newer commit found on $CURRENT_BRANCH branch. Skipping build."
-            echo "skip_build=true" >> $GITHUB_OUTPUT
+    - name: Download apps-to-build artifact
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)